Detectors/ITSMFT/ITS/tracking/src/VertexerTraits.cxx

```diff
@@ -374,6 +374,19 @@ void VertexerTraits::computeVertices()
         }
       }
     }
+    if (mVrtParams.allowSingleContribClusters) {
+      auto beamLine = Line{{mTimeFrame->getBeamX(), mTimeFrame->getBeamY(), -50.f}, {mTimeFrame->getBeamX(), mTimeFrame->getBeamY(), 50.f}}; // use beam position as contributor
```

Comment:
I don't see where do you see the beam position, should not you read the MeanVertex object, like here: 
https://github.com/AliceO2Group/AliceO2/blob/c0375b3d61d15e0ca7a4fc0da82af9dcdfd41696/Detectors/GlobalTrackingWorkflow/src/PrimaryVertexingSpec.cxx#L240
https://github.com/AliceO2Group/AliceO2/blob/c0375b3d61d15e0ca7a4fc0da82af9dcdfd41696/Detectors/GlobalTrackingWorkflow/src/PrimaryVertexingSpec.cxx#L180-L183
https://github.com/AliceO2Group/AliceO2/blob/c0375b3d61d15e0ca7a4fc0da82af9dcdfd41696/Detectors/GlobalTrackingWorkflow/src/PrimaryVertexingSpec.cxx#L211
etc...

Reply:
In ITS reconstruction we keep track of a rough beam position estimation by using previous vertices transverse positions, weighted on they contributors.
This is reset at every timeframe and have been reliable enough so far...
Making the seeding depending from global reconstruction might be an overkill.

Reply:
Does this mean that the beginning of every TF uses 0,0 mean vertex which progressively goes to its correct value?

Reply:
Yes, but after finding the first seeding vertex, usually we have a sufficient resolution for the tracker tolerances.
With Max we are also planning to make this a persistent information, but will require some additional work to make the TimeFrame object not to be reset at each load, like it is currently done instead.

Reply:
Well, I don't see why not using the MeanVertex, at least in the async mode, which is calibrated exactly for such cases. With current approach the very low intensity data, e.g. lhc22s PbPb will always use 0/0.

Reply:
I see your point. I can implement a flag to enable the usage of meanvertex in case global reconstruction is available, being sync/async.
Not sure about the timescale, depends on when we are going to start apass3.

I would anyways keep also the current option as the default so to be able not to depend from all the rest of the reconstruction whenever we want run ITS reco workflow.

Reply:
Agree, optionally one could require constraining by own previous vertices

Reply:
ok, there is still something I'm missing, this does not work yet after new changes. Do not merge it, pls.